zcash_primitives/CHANGELOG.md

```diff
@@ -15,6 +15,14 @@ and this library adheres to Rust's notion of
 - `sapling::SAPLING_COMMITMENT_TREE_DEPTH_U8` has been removed; use
   `SAPLING_COMMITMENT_TREE_DEPTH` instead.
 - `merkle_tree::incremental::write_auth_fragment_v1`
+- `merkle_tree::CommitmentTree::write` and `merkle_tree::CommitmentTree::read`
+  have been replaced by `merkle_tree::write_commitment_tree` and
+  `merkle_tree::read_commitment_tree` respectively.
+- `merkle_tree::IncrementalWitness::write` and `merkle_tree::IncrementalWitness::read`
+  have been replaced by `merkle_tree::write_incremental_witness` and
+  `merkle_tree::read_incremental_witness` respectively.
+- `merkle_tree::MerklePath::from_slice` has been replaced by 
+  `merkle_tree::merkle_path_from_slice`
```

Comment:
This is somewhat addressed by a later commit, but it would still be nice to reduce the verbosity of the changelog. Non-blocking, I'll do it during the next release PR if it isn't done here.